benchmark/http/upgrade.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const common = require('../common.js');
+const PORT = common.PORT;
+const net = require('net');
+
+const bench = common.createBenchmark(main, {
+  n: [5, 1000]
+});
+
+const reqData = 'GET / HTTP/1.1\r\n' +
+                'Upgrade: WebSocket\r\n' +
+                'Connection: Upgrade\r\n' +
+                '\r\n' +
+                'WjN}|M(6';
+
+const resData = 'HTTP/1.1 101 Web Socket Protocol Handshake\r\n' +
+                'Upgrade: WebSocket\r\n' +
+                'Connection: Upgrade\r\n' +
+                '\r\n\r\n';
+
+function main(conf) {
+  process.env.PORT = PORT;
+  var server = require('../fixtures/simple-http-server.js')
+  .listen(process.env.PORT || common.PORT)
```

Comment:
I am not sure I can follow `process.env.PORT = PORT` here. The only place I find that could modify the environment variable is running the benchmark if it passed in while running the benchmark but why do we want to protect against this? No-one should pass in that env anyways?

Reply:
This was copied over from other benchmarks. `http/simple` does this as well.

Reply:
That might be the case but I would argue it should be changed there as well instead of copying that part over as it is a no-op.